src/index.js

```diff
@@ -401,6 +574,32 @@ function drawPrefectureTable(prefectures, totals) {
   dataTableFoot.innerHTML = "<tr class='totals'><td>" + totalStr + "</td><td>" + totals.confirmed + "</td><td>" + totals.recovered + "</td><td>" + totals.deceased + "</td></tr>"
 }
 
+function drawTravelRestrictions() {
+  travelRestrictionsHelper('#banned-entry', ddb.travelRestrictions.japan.banned);
+  travelRestrictionsHelper('#visa-required', ddb.travelRestrictions.japan.visaRequired);
+  travelRestrictionsHelper('#self-quarantine', ddb.travelRestrictions.japan.selfQuarantine);
+  travelRestrictionsHelper('#other-restrictions', ddb.travelRestrictions.japan.other);
+
+  travelRestrictionsHelper('#foreign-banned-entry', ddb.travelRestrictions.foreignBorders.banned);
+  travelRestrictionsHelper('#foreign-visa-required', ddb.travelRestrictions.foreignBorders.visaRequired);
+  travelRestrictionsHelper('#foreign-self-quarantine', ddb.travelRestrictions.foreignBorders.selfQuarantine);
+  travelRestrictionsHelper('#foreign-other-restrictions', ddb.travelRestrictions.foreignBorders.other);
+}
+
+function travelRestrictionsHelper(elementId, countries) {
+  let countryList = [];
+  // Iterate through and render country links
+  _.orderBy(countries, 'name', 'desc').map(function(country){
+    let name = (LANG == 'en') ? country.name : country.nameJa;
+
+    countryList.unshift(`${country.emoji}<a href="${country.link}">${name}</a>`);
```

Comment:
```suggestion
    countryList.unshift(`<a href="${country.link}">${country.emoji} ${name}</a>`);
```
Making the flag clickable

Reply:
Sure, took the space out to prevent odd linebreaks that separate the flag and country name.